### DISCUSSION TITLE
Change Time Format

### DISCUSSION THREAD

**Discussion by gand07** [bellrichm/weewx-jas#122] [Q&A] [answered]
Hi
Do you use these from weewx.conf for the Time Format
hour = %H:%M
day = %X
week = %X (%A)
month = %x %X
year = %x %X
rainyear = %x %X
current = %x %X

I want to change
mm/dd/yyyy to dd/mm/yyyy

Can Insert the Time Format into Pages, Sections, etc

Gert

**Comment by bellrichm** [comment, marked as answer]
Since all formatting is done in Javascript, I unfortunately had to take a different approach. I also made it a bit more context sensitive. 

For the different date/time contexts, look here.
https://github.com/bellrichm/weewx-jas/blob/cc5abd7a2dc7e72d64b1fb19bda55ee9dc2165bd/skins/jas/lang/en.conf#L247

Here is a bit of information on where to update weewx.conf.
https://github.com/bellrichm/weewx-jas/wiki/Localization#dates-and-times

Please post back with any questions, clarifications, etc. It will help me to improve the documentation.
